ipalib/certmapping.py

```diff
@@ -0,0 +1,337 @@
+#
+# Copyright (C) 2016  FreeIPA Contributors see COPYING for license
+#
+
+import collections
+import jinja2
+import jinja2.ext
+import jinja2.sandbox
+import json
+import os.path
+import traceback
+
+from ipalib import errors
+from ipalib.text import _
+from ipapython.ipa_log_manager import root_logger
```

Comment:
Right, this is not used often in our code, but we try to follow common best practices in new code, and the best practice for logging seems to be to use module-specific loggers.